Soccar [Client]/src/nl/soccar/ui/fx/controller/CreateRoomFXMLController.java

```diff
@@ -7,21 +7,72 @@
 
 import java.net.URL;
 import java.util.ResourceBundle;
+import javafx.fxml.FXML;
 import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.Slider;
+import javafx.scene.control.TextField;
+import nl.soccar.library.Session;
+import nl.soccar.library.SessionController;
+import nl.soccar.library.Soccar;
+import nl.soccar.ui.Main;
+import nl.soccar.ui.fx.FXMLConstants;
 
 /**
  * FXML Controller class
  *
  * @author Luuk
  */
 public class CreateRoomFXMLController implements Initializable {
-
+    
+    @FXML
+    private Button btnLogOut;
+    @FXML
+    private Button btnCancel;
+    @FXML
+    private Button btnCreateRoom;
+    @FXML
+    private Label lblUsername;
+    @FXML
+    private TextField textFieldRoomName; 
+    @FXML
+    private TextField textFieldPassword; 
+    @FXML
+    private Slider sliderCapacity;
     /**
      * Initializes the controller class.
      */
+    
     @Override
     public void initialize(URL url, ResourceBundle rb) {
-        // TODO
+        btnLogOut.setOnAction(e -> {
+            Main.getInstance().logOut();
+        });
+        btnCancel.setOnAction(e -> {
+            Main.getInstance().setScene(FXMLConstants.LOCATION_MAIN_MENU);
+        });
+        btnCreateRoom.setOnAction(e -> {
+           if(!textFieldRoomName.getText().isEmpty()) {
+               createRoom();
+           } else {
+               textFieldRoomName.setStyle("-fx-text-box-border: red ; -fx-focus-color: red ;");
+           }
+        });
+        
+        lblUsername.setText(Soccar.getInstance().getCurrentPlayer().getUsername()); 
     }    
     
+    public void createRoom() {
+        String password;
+         
+        if (textFieldPassword.getText().isEmpty()) {
+            password = null;

```

Comment:
Redundant, password is al null. Invert deze if-statement door de ! operator te gebruiken (if (!textFieldPassword....).